modules/internal/localeModels/gpu/LocaleModel.chpl

```diff
@@ -299,12 +259,17 @@ module LocaleModel {
       halt("requesting a child from a GPULocale locale");
       return new locale(this);
     }
+
+    override proc isGpu() : bool { return true; }
   }
 
   const chpl_emptyLocaleSpace: domain(1) = {1..0};
   pragma "unsafe"
   const chpl_emptyLocales: [chpl_emptyLocaleSpace] locale;
 
+  pragma "unsafe"
+  const chpl_emptyLocalesModels: [chpl_emptyLocaleSpace] unmanaged AbstractLocaleModel;
```

Comment:
Is this used anywhere?

Reply:
Whoops, no this is cruft, I'll remove it.